src/Microsoft.Data.Sqlite.Core/SqliteConnection.cs

```diff
@@ -265,6 +265,30 @@ protected override DbCommand CreateDbCommand()
             => CreateCommand();
 
         /// <summary>
+        /// Create custom collation.
+        /// </summary>
+        /// <param name="name">Name of the collation.</param>
+        /// <param name="comparison">Method that compares two strings.</param>
+        public virtual void CreateCollation(string name, Comparison<string> comparison)
+            => CreateSqlite3Collation(name, null, (_, s1, s2) => comparison(s1, s2));
```

Comment:
Personally, I'd just call ` CreateCollation<object>(name, null, `... and inline `CreateSqlite3Collation` into the other overload.